app/views/admin/reservations/_profile.html.erb

```diff
@@ -42,13 +42,14 @@
   <div class="column col-sm-12 col-8">
 
     <% if Reservation.statuses.values_at(:requested, :approved, :building).include?(@reservation.status) %>
-      <%= render partial: "reservation_policies_warnings", locals: {reservation: @reservation} %>
+      <%= render partial: "admin/reservations/reservation_policies_warnings", locals: {reservation: @reservation} %>
```

Comment:
I got an error at some point switching between the tabs since this partial didn't have a (more) absolute path.